package.json

```diff
@@ -10,7 +10,7 @@
         "@material-ui/core": "^4.9.4",
         "@material-ui/icons": "^4.9.1",
         "@material-ui/lab": "^4.0.0-alpha.56",
-        "@plasm/types": "^1.4.0",
+        "@plasm/types": "^1.5.4-3",
```

Comment:
I stumbled upon to some problems after conversion. It will take some time for me to resolve.

Reply:
I came up to the following issue after upgrading to latest @polkadot/api
![image](https://user-images.githubusercontent.com/8452361/128321371-d3d25820-354e-4c54-b0ba-25f4bd0fc255.png)
Since there are overload accepting single argument I tried with upgrading TypeScript. TS upgrade to v4.0.2 solved the issue but introduced a new issue with eslint
![image](https://user-images.githubusercontent.com/8452361/128322137-3e5d6f21-2e10-4b1d-80a1-157fcbbbd25b.png)
I tried upgrading eslint, but I got an error that application is not supposed to run with this version and that manually increasing eslint version can lead to unexpected behavior. @hoonsubin do you have an idea how I can upgrade eslint?

Reply:
hmmm, seems like there was a breaking change in the package. Since the lockdrop module is exclusively used for main net Plasm and only Dusty requires the latest typings. So instead of upgrading the API package, I think removing the use of Dusty and `@plasm/types` entirely might be better.

This is related to issue https://github.com/PlasmNetwork/lockdrop-ui/issues/171

Reply:
Great. Thank you @hoonsubin. If I understood correctly, everything related to Dusty and Ropsten should go away from the project.

Reply:
Yep. That way, we don't need the latest polkadot api